third_party/pcsc-lite/naclport/server/src/google_smart_card_pcsc_lite_server/global.h

```diff
@@ -38,6 +43,30 @@ namespace google_smart_card {
 // already been initialized.
 void InitializeAndRunPcscLiteServer();
 
+class PcscLiteServerGlobal final {
+ public:
+  explicit PcscLiteServerGlobal(pp::Instance* pp_instance);
+  ~PcscLiteServerGlobal();
+  void Detach();
```

Comment:
Please write a comment for this function, because this is a nontrivial functionality.
(You may take as an example the existing docs for the Detach methods in other classes.)